core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -21,6 +21,15 @@
   <div class="protractor-test-conversation-input"
        angular-html-bind="activeCard.interactionHtml">
   </div>
+    <div ng-if="currentInteractionHints.length" style="text-align: right; padding: 8px">
+      <span tooltip="<[tooltipText]>">
```

Comment:
Perhaps pull out into a subdirective that's used by both the supplemental and tutor cards.